deploy/crs/v2/snippets/common_config.yaml

```diff
@@ -12,3 +12,7 @@ spec:
     # By default is the cr.Name
     applicationName: other
     adminPassword: example
+    # startupStrategy could be OpenShiftStartupStrategy or ControllerBasedStartupStrategy
+    startupStrategy:
+      strategyName: "OpenShiftStartupStrategy"
```

Comment:
Use controller as OSSS is the defualt

Reply:
Ok

Reply:
Please, leave there OpenShiftStartupStrategy as default one. This strategy is used as default so far and we should change, as it may affect running apps during migration.